<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>233</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 225, 227);</string>
  </property>
  <widget class="QPushButton" name="sumbutton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 203, 213);</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="subbutton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 203, 213);</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="divbutton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 203, 213);</string>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mulbutton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 203, 213);</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 93, 158);</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 93, 158);</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 93, 158);</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 93, 158);</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button6">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 93, 158);</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 93, 158);</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 93, 158);</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button8">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 93, 158);</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button9">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 93, 158);</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="equal_button">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 203, 213);</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button0">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 93, 158);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>AC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sin_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 43, 128);</string>
   </property>
   <property name="text">
    <string>sin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cos_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 43, 128);</string>
   </property>
   <property name="text">
    <string>cos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tan_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 43, 128);</string>
   </property>
   <property name="text">
    <string>tan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cot_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 43, 128);</string>
   </property>
   <property name="text">
    <string>cot</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sqrt_button">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(217, 217, 217);</string>
   </property>
   <property name="text">
    <string>sqrt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="float_button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(217, 217, 217);</string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="log_button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(217, 217, 217);</string>
   </property>
   <property name="text">
    <string>log</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pn_button">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(217, 217, 217);</string>
   </property>
   <property name="text">
    <string>+/-</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textbox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="fact_button">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(217, 217, 217);</string>
   </property>
   <property name="text">
    <string>fact</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
